src/components/PrivacyDisclaimer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+
+const Footer = styled.div`
+  width: 100vw;
+  overflow: hidden;
+  padding: 12px 40px;
+  display: flex;
+  border-top: 1px solid #efefef;
+  background-color: #fff;
```

Comment:
This makes it easy to oversee/ignore the warning

Reply:
Yea was kinda intentional, people really hated the yellow tag

Reply:
I think so, making it more visible and removing the 30 sec hiding will force user to make a choice. It might be slightly annoying but it's a one time user interaction that we need to do the tracking we want. It's also something that everybody have been conditioned to do the last 3-4(?) years because of the widespread use of cookie disclaimers.

Reply:
Fair I'll switch to a nicer color :P